src/expr/src/vector_op/agg/approx_count_distinct.rs

```diff
@@ -28,7 +28,7 @@ const COUNT_BITS: u8 = 64 - INDEX_BITS; // number of non-index bits in each 64-b
 
 // Approximation for bias correction for 16384 registers. See "HyperLogLog: the analysis of a
 // near-optimal cardinality estimation algorithm" by Philippe Flajolet et al.
-const BIAS_CORRECTION: f64 = 0.72125;
+const BIAS_CORRECTION: f64 = 0.7213 / (1. + (1.079 / NUM_OF_REGISTERS as f64));
```

Comment:
Good catch, we will try to be consistent here and use the slightly more accurate formula

Reply:
More importantly, formula can help us change `BIAS_CORRECTION ` when `NUM_OF_REGISTERS ` is changed.